store/src/java/com/zimbra/cs/html/owasp/OwaspHtmlSanitizer.java

```diff
@@ -79,6 +81,52 @@ private String checkUnbalancedTags(String html) {
         }
         return html;
     }
+    
+    /** 
+     * A method to check & remove any any child parent comparator inside <style> tag 
+     * 
+     * @return the Formatted Html after removing child comparators(>)
+     */
+    private String removeChildComaparator(String html) {
+        if (StringUtil.isNullOrEmpty(html)) {
+            return html;
+        }
+        ArrayList<String> cssTagsList = new ArrayList<String>();
+        // in cssTagsList array we can add multiple tags 
+        cssTagsList.add("tbody");
+        cssTagsList.add("tr");
+        cssTagsList.add("div");
```

Comment:
all tags should be case insensitive.
related to: lines 96-98, 100-102, 104-105 and 110-111

Reply:
added uppercase & firstletter uppercase for all tags ... ( style,tbody,tr,div)

Reply:
it should be implemented using regex or something.
all patterns are valid as html: `TBODY, TBODy, TBOdy, TBody, Tbody, tbody`

Reply:
or find a way to execute `split` or `contains` with case insensitive.

Reply:
Added regex for split method & for contain method I have added toLowercase method to check condition only